lib/Itho/IthoCC1101.cpp

```diff
@@ -0,0 +1,852 @@
+/*
+   Author: Klusjesman, supersjimmie, modified and reworked by arjenhiemstra
+*/
+#define DEBUG 0
+
+#define BYTE_TO_BINARY_PATTERN "%c,%c,%c,%c,%c,%c,%c,%c,"
+#define BYTE_TO_BINARY(byte)  \
```

Comment:
`BYTE_TO_BINARY` and `BYTE_TO_BINARY_PATTERN` do not seem to be used.

Reply:
This is a remnant of development debug code, can be removed